src/olympia/amo/validators.py

```diff
@@ -74,3 +77,20 @@ class PreventPartialUpdateValidator:
     def __call__(self, value, serializer_field):
         if serializer_field.parent.partial:
             raise fields.SkipField()
+
+
+def HttpHttpsURLValidator(message=None, code=None):
+    return URLValidator(message=message, code=code, schemes=('http', 'https'))
+
+
+def NoAMODomainValidator():
+    return RegexValidator(
+        regex=r'%s' % re.escape(settings.DOMAIN),
```

Comment:
Using `settings.EXTERNAL_SITE_URL` seems safer then `settings.DOMAIN` - if was used on reviewers.m.o, services.a.m.o, or the admin instances it'd consistently be the primary addons.mozilla.org domain.

Reply:
Fair. We weren't consistent before, with the validator used in devhub using `DOMAIN` and the one in the API using `EXTERNAL_SITE_URL`. I've picked `DOMAIN` but I agree maybe `EXTERNAL_SITE_URL` is better here. I'll update.